tests/internal/e2e/test/specs/vcast.create_script_2_and_run.test.ts

```diff
@@ -170,6 +171,70 @@ describe("vTypeCheck VS Code Extension", () => {
     });
   });
 
+  it("should verify no coverage in status bar when opening quotes_example.cpp. ", async () => {
```

Comment:
clearer: when opening uninstrumented quotes_example.cpp

Reply:
Changed in a331feec08a94d4a0f35d05264aeacbad2967b4f .